### PR TITLE
Korean Translation

### PR DESCRIPTION
**Pull request by ScathachSkadi**
Added new strings and translated some. Will translate rest after I wake up.

### CODE REVIEW

File: app/src/main/res/values-ko/localized.xml [hunk 1]
```diff
@@ -142,8 +142,8 @@
     <string name="p_boost_item">부스트 아이템</string>
     <string name="p_brave_chains">브레이브 체인 (실험적)</string>
     <string name="p_rearrange_cards">카드 재배열</string>
-    <string name="p_auto_skill_cmd_text">AutoSkill Cmd TextBox</string>
-    <string name="p_auto_skill_cmd_text_summary">Show text-box for editing AutoSkill commands</string>
+    <string name="p_auto_skill_cmd_text">오토 스킬 Cmd 텍스트 박스</string>
```
Comment (reconman) [inline app/src/main/res/values-ko/localized.xml:145]: Shouldn't `Cmd` also be translated as `커맨드를`?

File: app/src/main/res/values-ko/localized.xml [hunk 2]
```diff
@@ -159,23 +159,23 @@
 
     <string name="p_fine_tune">미세 조정</string>
     <string name="p_fine_tune_default">기본값: %d</string>
-    <string name="p_fine_tune_support">Support</string>
+    <string name="p_fine_tune_support">서포트</string>
     <string name="p_fine_tune_support_swipes_per_update">Swipes per Update</string>
```
Comment (reconman) [inline app/src/main/res/values-ko/localized.xml:163]: Not translated

File: app/src/main/res/values-ko/localized.xml (hunk 1 above)
Comment (ScathachSkadi) [reply app/src/main/res/values-ko/localized.xml:145]: Oh that was the short form of command. Right. Will change that and also what does b/w mean? Black and white?

Comment (reconman) [reply app/src/main/res/values-ko/localized.xml:145]: No, in this context it's between. I've never heard of the abbreviation myself, though.

Comment (ScathachSkadi) [reply app/src/main/res/values-ko/localized.xml]: Me neither. Thanks for that.

**Comment by ScathachSkadi** [comment]
https://del.dog/imeappocop.txt

Crashes when you try to edit the command of auto skill

**Comment by reconman** [comment]
@MathewSachin Seems like this error is caused by https://github.com/MathewSachin/Fate-Grand-Automata/blob/master/app/src/main/java/com/mathewsachin/fategrandautomata/ui/auto_skill_maker/AutoSkillMakerViewModel.kt#L373

See https://stackoverflow.com/a/59633268/5971497

I tried renaming it to `clearCommand` in Android Studio, but now I'm getting compiler errors, it can't resolve `AutoSkillMakerActivityArgs`.

File: app/src/main/res/values-ko/localized.xml (hunk 1 above)
Comment (MathewSachin) [reply app/src/main/res/values-ko/localized.xml]: Reworded `AutoSkill Cmd TextBox` to `Skill Command TextBox`.
If I put `between` in place of `b/w` the text becomes too long. @reconman Should I reword `Delay between repeated clicks` to `Repeat click delay`?

**Comment by MathewSachin** [comment]
@reconman I renamed the clear function and tried a release build. It seems like the problem is something else.

This is happening only in release builds, so, Proguard is removing something again.

> it can't resolve AutoSkillMakerActivityArgs.

It'll resolve once you build. It's compile-time generated code.

**Comment by MathewSachin** [comment]
@reconman You were right. My bad. It was a permission problem on my phone.

**Comment by MathewSachin** [comment]
@ScathachSkadi @reconman I've confirmed that the crash is fixed in the latest GitHub Actions build for this PR

**Comment by ScathachSkadi** [comment]
Awesome.

**Comment by ScathachSkadi** [comment]
@MathewSachin is there something that I need to retranslate or it is the same as before?

**Comment by MathewSachin** [comment]
@ScathachSkadi Nothing new to translate. Just check that all screens render properly, in case I missed something.

File: app/src/main/res/values-ko/localized.xml (hunk 1 above)
Comment (reconman) [reply app/src/main/res/values-ko/localized.xml]: Maybe `Time between clicks`

Comment (MathewSachin) [reply app/src/main/res/values-ko/localized.xml]: We need to make it clear that it is between repeated clicks somehow.

Comment (reconman) [reply app/src/main/res/values-ko/localized.xml]: @ScathachSkadi We decided on Multi-Click Interval for the English version.

Comment (ScathachSkadi) [reply app/src/main/res/values-ko/localized.xml]: Well I think the translation itself would change in that case

**Comment by ScathachSkadi** [comment]
https://ibb.co/xKZkSQK
I don't think Gold x0 should be seen here. The rest of the app has no problems so far said the people who I gave the forked apk

**Comment by MathewSachin** [comment]
You'll have to live with that `Gold x0`. It just prints the `enum` value as it is.

**Comment by ScathachSkadi** [comment]
Oh okay. People are just starting to test it out so I will let people use the app a bit more to see if there's something wrong with the translation. I am going to try the app irl myself to kill some Hassans.

**Comment by MathewSachin** [comment]
I guess I'll merge this now. You can open another PR if you want to make more changes.

**Pull request merged**